.github/workflows/ci.yaml

```diff
@@ -57,13 +57,31 @@ jobs:
     - name: Test funcx-endpoint by just importing
       run: |
         cd funcx_endpoint
-        pip install .
-        python -c "from funcx.endpoint.version import VERSION"
+        pip install -e .
```

Comment:
Any reason why we the editable flag is added? I don't think this affects the user.